test/integration/consul-container/test/resource/resource_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
```

Comment:
Maybe `grpc_forwarding_test.go` would be more descriptive given the current contents?

Reply:
Yup, sounds better, updated